hooks/use-css-vars/src/generate-tokens-map.ts

```diff
@@ -0,0 +1,35 @@
+import type { Theme, Value } from './use-css-vars.types';
+
+const isObject = (value: Value) =>
```

Comment:
nit: We could probably also break these utils into a util file or package since they're pretty generic.

Reply:
Speaking to isObject and isEmptyObject. GitHub removed my highlighting...

Reply:
Yeah, I feel like there must be a library somewhere I can pull from. I am *pretty* sure I am not the first one who needed an `isObject` / `isEmptyObject` util.

Reply:
I found this one in Lodash: [isPlainObject](https://github.com/lodash/lodash/blob/ddfd9b11a0126db2302cb70ec9973b66baec0975/lodash.js#L12036). 🤷‍♀️

Or I could create a note to extract it into its own package withing Blueprints?

What do you think?

Reply:
I'm ok with either, as long as the Lodash one is actually maintained. I've been burned by that lib lately when the maintainer took off for a year and we had security bugs because of it. This feels somewhat low risk though. I am ok doing this in another PR though so it's not a blocker to me.

Reply:
Yeah, I am suspicious.